aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -216,7 +245,7 @@ while [ $(healthcheck) = "200" ]; do
 	sleep 300
 done
 
-MESSAGE="Profile ($EXISTING_PROFILE) or rule ($EXISTING_RULE)"
+MESSAGE="Profile ($EXISTING_PROFILE) or rule ($EXISTING_RULE) or integration ($EXISTING_QUALYS)"
```

Comment:
`EXISTING_QUALYS` is optional, so please copy the artifactory example from line 251 below